tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/TinkerGraphFeatureTest.java

```diff
@@ -33,7 +33,11 @@
         tags = "not @RemoteOnly and not @GraphComputerOnly and not @AllowNullPropertyValues",
         glue = { "org.apache.tinkerpop.gremlin.features" },
         objectFactory = TinkerGraphFeatureTest.TinkerGraphGuiceFactory.class,
-        features = { "classpath:/org/apache/tinkerpop/gremlin/test/features" },
+        features = { "classpath:/org/apache/tinkerpop/gremlin/test/features"
```

Comment:
i assume there is some clean-up still to do? why was the `TinkerMergeEVStepStrategy` removed with its related steps? is bringing those back part of the clean-up?

Reply:
Don't need them anymore. Merge steps create a Traversal now to search for the Map constraints. That traversal will be optimized to use the TinkerGraph indices when run against a TG.

Reply:
But yes I will remove these two lines before merging.

Reply:
> Merge steps create a Traversal now to search for the Map constraints.

ok - i must have missed that in my first pass. that part of the diff was a bit dense. thanks.